db/migrate/20150120095012_create_categories.rb

```diff
@@ -2,7 +2,7 @@ class CreateCategories < ActiveRecord::Migration
   def change
     create_table :categories do |t|
       t.string :name
-      t.string :link
+      t.string :simplified_name
       
```

Comment:
To make your queries more efficient you could add an index on the simplified_name. Has to be set after creating the table:
```
def change
  create_table :categories do |t|
    ...
  end
  add_index :categories, :simplified_name
end
```
That way you'll be able to do search for a category directly from its simplified_name, rather than the id